File: java/AnggotaDPR.java
public class AnggotaDPR {
    private int id;
    private String nama;
    private String bidang;
    private String partai;

    public AnggotaDPR(int id, String nama, String bidang, String partai) {
        this.id = id;
        this.nama = nama;
        this.bidang = bidang;
        this.partai = partai;
    }

    public AnggotaDPR() {
        this.id = 0;
        this.nama = "";
        this.bidang = "";
        this.partai = "";
    }

    public void setId(int id) {
        this.id = id;
    }

    public void setNama(String nama) {
        this.nama = nama;
    }

    public void setBidang(String bidang) {
        this.bidang = bidang;
    }

    public void setPartai(String partai) {
        this.partai = partai;
    }

    public int ambilId() {
        return id;
    }

    public String ambilNama() {
        return nama;
    }

    public String ambilBidang() {
        return bidang;
    }

    public String ambilPartai() {
        return partai;
    }
}

File: cpp/AnggotaDPR.cpp
#include "AnggotaDPR.hpp"

AnggotaDPR::AnggotaDPR(unsigned int id, string nama, string bidang, string partai) {
    this->id = id;
    this->nama = nama;
    this->bidang = bidang;
    this->partai = partai;
}

AnggotaDPR::AnggotaDPR() {
    this->id = 0;
    this->nama = "";
    this->bidang = "";
    this->partai = "";
}

AnggotaDPR::~AnggotaDPR() {
}

void AnggotaDPR::setId(unsigned int id) {
    this->id = id;
}

void AnggotaDPR::setNama(string nama) {
    this->nama = nama;
}

void AnggotaDPR::setBidang(string bidang) {
    this->bidang = bidang;
}

void AnggotaDPR::setPartai(string partai) {
    this->partai = partai;
}

unsigned int AnggotaDPR::ambilId() {
    return id;
}

string AnggotaDPR::ambilNama() {
    return nama;
}

string AnggotaDPR::ambilBidang() {
    return bidang;
}

string AnggotaDPR::ambilPartai() {
    return partai;
}